cli/src/delegator.ts

```diff
@@ -296,6 +298,20 @@ export function reporterListHandler() {
     try {
       const endpoint = buildUrl(ORAKL_NETWORK_DELEGATOR_URL, `reporter`)
       const result = (await axios.get(endpoint)).data
+
+      for (const reporter of result) {
+        if (!reporter.contract) {
+          continue
+        }
+        const url = new URL(AGGREGATOR_ENDPOINT)
+        url.searchParams.append('address', reporter.contract[0])
+        const aggregatorResult = (await axios.get(url.toString())).data
+        console.log(aggregatorResult)
+        if (aggregatorResult && aggregatorResult.name) {
+          reporter.name = aggregatorResult.name
```

Comment:
If I understand it correctly, you are in-place modifying data inside of `result` which are later printed on line 315. We should avoid this side effect problem solving. Is there a way to do it without it?

Reply:
rather than adding to result data, each elements can be printed within iteration, how do you think?

Reply:
In that case, I think it is better to keep it as it is.